torch/_inductor/scheduler.py

```diff
@@ -2497,7 +2497,8 @@ def codegen(self):
 
     def is_unaligned_buffer(self, buf_name):
         if buf_name in V.graph.graph_inputs:
-            return not config.assume_aligned_inputs
+            # See Note: [Input Alignment handling in Inductor]
+            return buf_name not in V.graph.aligned_inputs
```

Comment:
I think you'll need to rebase past https://github.com/pytorch/pytorch/pull/124030/files#diff-34f851dde2ac5502be643489261bb0115d0cd873fac5367831f9850b471b3279R68